features/swizzling_disabled/automatic_spans.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Automatic instrumentation spans
+
+  Scenario: AutoInstrumentViewLoadScenario with swizzling disabled
+    Given I run "AutoInstrumentViewLoadScenario"
+    And I should receive no traces
```

Comment:
Tests that expect no traces are generally very weak.  E.g. if the app simply crashes then the test will still pass.  We might need to have a chat about how we might test this another way, but one idea could be to have the scenario sent a request to Maze Runners `/logs` endpoint and add a step to check for that.  At least we'd have some confidence that the app had not crashed.

Reply:
Noted. Swizzling_disabled is not in the original scope of this task, so for now I'm creating a ticket to improve the new e2e tests in the future.